modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/DdlWithMockedManagersTest.java

```diff
@@ -156,11 +163,56 @@ void before() throws NodeStoppingException {
         queryProc.start();
     }
 
+    /**
+     * Checks that appropriate methods called form table manager on rel node construction.
+     */
+    @Test
+    void checkAppropriateTableFound() throws Exception {
+        TableManager tableManager = mock(TableManager.class);
+
+        SqlSchemaManagerImpl schemaManager = new SqlSchemaManagerImpl(tableManager, () -> {});
+        AtomicBoolean tableByIdCalled = new AtomicBoolean();
+        IgniteUuid tblId = new IgniteUuid(UUID.randomUUID(), 0);
+
+        when(tableManager.table(any(IgniteUuid.class))).thenAnswer((k) -> {
```

Comment:
mockito has an API to verify whether particular method was called or not. Please see `org.mockito.Mockito#verify(T)`